vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -272,6 +272,12 @@ def invoke_fused_moe_kernel(A: torch.Tensor, B: torch.Tensor, C: torch.Tensor,
     )
 
 
+def maybe_set_triton_cache_manager(module: str) -> None:
+    cache_manger = os.environ.get("TRITON_CACHE_MANAGER", None)
+    if cache_manger != module:
+        os.environ["TRITON_CACHE_MANAGER"] = module
```

Comment:
If the user manually sets this env, can we modify it? Additionally, I suggest adding a log message for clarification

Reply:
have changed it so that we only set it if the user has not